test/suite/test_compact11.py

```diff
@@ -0,0 +1,197 @@
+            time.sleep(0.1)
+
+    def turn_off_bg_compact(self):
+        self.session.compact(None, f'background=false')
+        while self.get_bg_compaction_running():
+            time.sleep(1)
+
+    def compare_bitmap(self, orig, new):
+        # Compare the bitmaps from the metadata. Once a bit is set, it should never
+        # be cleared. But new bits could be set. So the check is only: if the original
+        # bitmap has a bit set then the current bitmap must be set for that bit also. 
+        #
+        # First convert both bitmaps to a binary string, accounting for any possible leading
+        # zeroes (that would be truncated off). Then compare bit by bit.
+        orig_bits = bin(int('1'+orig, 16))[3:]
+        new_bits = bin(int('1'+new, 16))[3:]
+        self.prout("Original bitmap in binary: " + orig_bits)
+        self.prout("Reopened bitmap in binary: " + new_bits)
+        for o_bit, n_bit in zip(orig_bits, new_bits):
+            if o_bit != '0':
+               self.assertTrue(n_bit != '0')
+
+    def parse_blkmods(self, uri):
+        meta_cursor = self.session.open_cursor('metadata:')
+        config = meta_cursor[uri]
+        meta_cursor.close()
+        # The search string looks like: ,blocks=feffff1f000000000000000000000000
+        # Obtain just the hex string.
+        b = re.search(',blocks=(\w+)', config)
+        self.assertTrue(b)
+        # The bitmap string after the = is in group 1.
+        blocks = b.group(1)
+        self.prout(f"parse_blkmods for '{uri}': {blocks}")
+        return blocks
+
+    def test_compact11(self):
+        # Create N tables.
+        uris = []
+        files = []
+        bitmaps = []
+        for i in range(self.num_tables):
+            uri = self.uri_prefix + f'_{i}'
+            uris.append(uri)
+            files.append(f'file:test_compact11_{i}.wt')
+            self.session.create(uri, self.create_params)
+
+        # Populate the first half of each table.
+        for uri in uris:
+            self.populate(uri, 0, self.table_numkv // 2, self.value_size)
+
+        # Write to disk.
+        self.session.checkpoint()
+
+        os.mkdir(self.backup_incr)
+        self.initial_backup = True
+        self.take_full_backup(self.backup_incr)
+        self.initial_backup = False
+
+        # Insert the latter 50% in each table.
+        for uri in uris:
+            self.populate(uri, self.table_numkv // 2, self.table_numkv, self.value_size)
+
+        # Write to disk.
+        self.session.checkpoint()
+
+        # Delete 50% of each file.
+        for uri in uris:
+            self.delete_range(uri, 50 * self.table_numkv // 100)
+
+        # Write to disk.
+        self.session.checkpoint()
+        for uri in files:
+            bitmaps.append(self.parse_blkmods(uri))
+
+        # Enable background compaction.
+        self.turn_on_bg_compact('free_space_target=1MB')
+
+        # Every time compaction makes progress, perform an incremental backup.
```

Comment:
We are not doing this anymore at all?

Reply:
Yes, there are a few reasons for the change. 
1.  How I understood the reason for the testing is to specifically test for how the incremental backup bitmap field is interacted with background compaction. If that is the case, we don't need multiple incremental backups to check this and only one.
2. Once a second incremental backup happens {{blk_parsemods}} can't be used to check if the bitmapping is correct and unfortunately the ordering of how we can read the {{blocks}} field is not ordered.

Reply:
I think having background compaction and incr backups in parallel is also an interesting scenario. If we try to keep this, could we compare the "just taken" incr backups straight away instead of waiting for the loop to finish? This way, we will not compare the nth iteration of an incr backup but the fresh new one.

Reply:
done! I have made those changes in compact10 [5138fc6](https://github.com/wiredtiger/wiredtiger/pull/9977/commits/5138fc6cc4875fa9b291fc03b618948222bb21df)